src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -8,8 +8,12 @@
 #include <cstdio>
 #include <iostream>
 #include <numeric>
+#include <transformations/utils/utils.hpp>
```

Comment:
why do we need it here?

Reply:
Because `has_op_with_type()` need this head file.